doc/nrf/protocols/wifi/advanced_modes/wifi_psa_support.rst

```diff
@@ -0,0 +1,37 @@
+.. _ug_nrf70_developing_wifi_psa_support:
+
+Wi-Fi PSA support
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series device Wi-Fi solution supports Platform Security Architecture(PSA) security framework.
+The nRF70 Series device Wi-Fi solution achieves this by providing PSA API support for Wi-Fi cryptographic operations.
```

Comment:
I think we can remove Wi-Fi solution since it is known that 70 Series is a Wi-Fi Series.
```suggestion
The nRF70 Series devices support the Platform Security Architecture (PSA) security framework.
This is achieved by providing PSA API support for Wi-Fi® cryptographic operations.
```